samples/snippets/src/main/java/com/example/mediatranslation/TranslateFromFile.java

```diff
@@ -0,0 +1,95 @@
+import com.google.cloud.mediatranslation.v1beta1.StreamingTranslateSpeechResult;
+import com.google.cloud.mediatranslation.v1beta1.TranslateSpeechConfig;
+import com.google.protobuf.ByteString;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class TranslateFromFile {
+
+  public static void translateFromFile() throws IOException {
+    // TODO(developer): Replace these variables before running the sample.
+    String filePath = "path/to/audio.raw";
+    translateFromFile(filePath);
+  }
+
+  public static void translateFromFile(String filePath) throws IOException {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
+    try (SpeechTranslationServiceClient client = SpeechTranslationServiceClient.create()) {
+      Path path = Paths.get(filePath);
+      byte[] content = Files.readAllBytes(path);
+      ByteString audioContent = ByteString.copyFrom(content);
+
+      TranslateSpeechConfig translateSpeechConfig =
+          TranslateSpeechConfig.newBuilder()
+              .setAudioEncoding("linear16")
+              .setSampleRateHertz(16000)
+              .setSourceLanguageCode("en-US")
+              .setTargetLanguageCode("fr-FR")
+              .build();
+
+      StreamingTranslateSpeechConfig config =
+          StreamingTranslateSpeechConfig.newBuilder().setAudioConfig(translateSpeechConfig).build();
+
+      BidiStream<StreamingTranslateSpeechRequest, StreamingTranslateSpeechResponse> bidiStream =
+          client.streamingTranslateSpeechCallable().call();
+
+      // The first request contains the configuration.
+      StreamingTranslateSpeechRequest requestConfig =
+          StreamingTranslateSpeechRequest.newBuilder().setStreamingConfig(config).build();
+
+      // The second request contains the audio
+      StreamingTranslateSpeechRequest request =
+          StreamingTranslateSpeechRequest.newBuilder().setAudioContent(audioContent).build();
+
+      bidiStream.send(requestConfig);
+      bidiStream.send(request);
+
+      for (StreamingTranslateSpeechResponse response : bidiStream) {
+        // Once the transcription settles, the response contains the
+        // is_final result. The other results will be for subsequent portions of
+        // the audio.
+        StreamingTranslateSpeechResult res = response.getResult();
+        String translation = res.getTextTranslationResult().getTranslation();
+        String source = res.getRecognitionResult();
+
+        if (res.getTextTranslationResult().getIsFinal()) {
+          System.out.println(String.format("\nFinal translation: %s", translation));
+          System.out.println(String.format("Final recognition result: %s", source));
+          break;
```

Comment:
If the `break` is removed and this is turned to a `else` statement, will the program exit?

Reply:
No it wont exit, it throw OUT_OF_RANGE: Timeout between requests `OutOfRangeException`